src/main/java/com/google/cloud/tools/gradle/endpoints/framework/client/task/ExtractDiscoveryDocZipsTask.java

```diff
@@ -55,7 +55,7 @@ public void setDiscoveryDocsDir(File discoveryDocsDir) {
   @TaskAction
   public void extractDiscoveryDocs() {
     getProject().delete(discoveryDocsDir);
-    discoveryDocsDir.mkdirs();
+    getProject().mkdir(discoveryDocsDir);
```

Comment:
right, so project.mkdir I think throws its own errors. It works differently than mkdirs.